core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/LDAPConnectionContext.java

```diff
@@ -675,4 +824,21 @@ private String createFormattedLog(List<String> logPropertiesList) {
         }
         return sb.toString();
     }
+
+    /**
+     * Convert retry waiting time string to long.
+     *
+     * @param retryWaitingTime
+     * @return
```

Comment:
Add description for return

Reply:
Addressed with https://github.com/wso2/carbon-kernel/pull/2272